src/Generators/Directory.Build.targets

```diff
@@ -9,5 +9,9 @@
   <PropertyGroup>
     <!-- Stop complaining about enabling /doc for tests -->
     <NoWarn>$(NoWarn);RT0000</NoWarn>
+
+    <!-- Stop warnings about banned symbols in generators -->
+    <!-- https://github.com/dotnet/roslyn/issues/63290 -->
+    <NoWarn>$(NoWarn);RS1035</NoWarn>
```

Comment:
The report generators use IO, so we either need to do it differently or keep this suppression.

Reply:
Does it affect all gen projects? -or- Can we suppress only in the affected project?

Reply:
This would affect all gen projects. The suppression can be moved. 👍